sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/test/java/com/azure/spring/sample/eventhubs/binder/EventHubBinderApplicationIT.java

```diff
@@ -24,7 +24,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = EventHubBinderApplication.class)
 @AutoConfigureMockMvc
-@TestPropertySource(locations = "classpath:application-test.properties")
+@ActiveProfiles("manual")
```

Comment:
I thought we are going to make another PR for the integration tests?

Reply:
Reverted.